core/src/main/scala/org/apache/spark/shuffle/ShuffleBlockPusher.scala

```diff
@@ -361,7 +366,8 @@ private[spark] class ShuffleBlockPusher(conf: SparkConf) extends Logging {
     for (reduceId <- 0 until numPartitions) {
       val blockSize = partitionLengths(reduceId)
       logDebug(
-        s"Block ${ShufflePushBlockId(shuffleId, partitionId, reduceId)} is of size $blockSize")
+        s"Block ${ShufflePushBlockId(shuffleId, partitionId, shuffleMergeId,
```

Comment:
change parameter order : shuffleId, shuffleMergeId, partitionId.

Please check the entire PR very carefully to avoid these kinds of errors; I might have missed some.

Reply:
Again I missed to change the order here as part of the refactor. I carefully looked into the entire PR to make sure there are no issues like this elsewhere. Thanks for pointing out.